src/server/html.js

```diff
@@ -13,10 +14,20 @@ export default (config, assets, store, headers, component) => {
     );
   }
 
-  return '<!doctype html>\n' + ReactDOM.renderToString(
-    <html lang="en-us">
-      <Head additions={config.html.head} assets={assets} store={store} headers={headers} />
-      <Body assets={assets} store={store} headers={headers} component={component} />
+  const header = ReactDOM.renderToString(<Head additions={config.html.head} assets={assets} store={store} headers={headers} />);
+  const body = ReactDOM.renderToString(<Body assets={assets} store={store} headers={headers} component={component} />);
+  const head = Helmet.rewind();
+
+  return `
+    <!doctype html>
+    <html ${head.htmlAttributes.toString()}>
+        <head>
+            ${header.replace('<head>', '').replace('</head>', '')}
```

Comment:
This is using a template literal (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) - was just copying the example https://github.com/nfl/react-helmet#as-string-output